src/components/Calculator.css

```diff
@@ -0,0 +1,50 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.output {
+  width: 400px;
+  height: 30px;
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
- [ ] [OPTIONAL] Let's remove the `margin-bottom` in order for your project to look like the design template provided and let’s align the 0 to the right instead of centering it.

###Your Design
<img width="822" alt="Screenshot 2023-01-10 at 23 48 10" src="https://user-images.githubusercontent.com/58692590/211679733-51f03aef-7574-484e-9874-f3d1cff5a5b7.png">
### Design Template
<img width="289" alt="Screenshot 2023-01-10 at 23 50 02" src="https://user-images.githubusercontent.com/58692590/211679758-99e9b69c-bfc6-4dc2-a903-4dc3014969bd.png">